src/main/java/dev/symphony/harmony/mixin/food/GlowBerryMixin.java

```diff
@@ -0,0 +1,30 @@
+package dev.symphony.harmony.mixin.food;
+
+import dev.symphony.harmony.config.HarmonyConfig;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.entity.effect.StatusEffects;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(Item.class)
+public class GlowBerryMixin {
```

Comment:
Add Feature description and credits as comments to the class. (use https://github.com/Team-Symphony/Harmony/blob/0.0.1-dev%2B1.21.0/src/main/java/dev/symphony/harmony/mixin/transportation/VehiclesMoveThroughLeaves.java as reference)

Reply:
done